index.html

```diff
@@ -1005,55 +1005,84 @@ <h2>Continue Watching & Play Next</h2>
         </code>
       </pre>
     </div>
+  </section>
+
+  <section id="play-next-tv-episodes">
+    <h2>Play Next and TV Episodes</h2>
 
     <p>
-      A user agent might want to display the next episode in a TV series to play
-      next. In this case, the user agent should calculate the
-      <dfn>play next episode</dfn> using the following algorithm:
+      A <a>media feed item</a> that is a TV series can have up to two TV
+      episodes embeded in it. These can be embedded directly in the
```

Comment:
nit: embedded*
and "if there are more than two"

Reply:
At the moment the current episode and play next episode algorithms will silently drop any extra episodes. I think this is the better solution because I can see us in the future having support for more episodes and having some sort of picker in the UI

Reply:
Hm...? This comment was just about grammar and spelling. I think this reply goes with a different comment. :)

Reply:
My bad.